roxctl/common/client.go

```diff
@@ -111,9 +111,20 @@ func (client *roxctlClientImpl) DoReqAndVerifyStatusCode(path string, method str
 	return resp, nil
 }
 
+func sanitizeHeaderValue(value string) string {
+	const whitelist = `abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.,:;+-_=/*{}[]'" `
```

Comment:
Please leave a note why you're suggesting these chars. Quickly scanning https://www.rfc-editor.org/rfc/rfc7230#section-3.2.6 makes me wonder whether `{}` should be on the list so clarification will help.

Reply:
Thanks for noticing that. I improved the logic, and added a comment. `{}` should be on the list, btw.